services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/volumes_resolver.py

```diff
@@ -2,6 +2,66 @@
 from pathlib import Path
 from typing import Any, Dict
 
+from settings_library.rclone import RCloneSettings, S3Provider
+from models_library.projects_nodes_io import NodeID
+from models_library.projects import ProjectID
+
+from .errors import DynamicSidecarError
+
+
+def _get_s3_volume_driver_config(
+    r_clone_settings: RCloneSettings,
+    project_id: ProjectID,
+    node_uuid: NodeID,
+    volume_name: str,
+) -> Dict[str, Any]:
+    driver_config = {
+        "Name": "rclone",
+        "Options": {
+            "type": "s3",
+            "s3-access_key_id": r_clone_settings.S3_ACCESS_KEY,
+            "s3-secret_access_key": r_clone_settings.S3_SECRET_KEY,
+            "s3-endpoint": r_clone_settings.endpoint_url,
+            "path": f"{r_clone_settings.S3_BUCKET_NAME}/{project_id}/{node_uuid}/{volume_name}",
+            "allow-other": "true",
+            "vfs-cache-mode": "full",
+            # Directly connected to how much time it takes for
+            # files to appear on remote s3, please se discussion
+            # SEE https://forum.rclone.org/t/file-added-to-s3-on-one-machine-not-visible-on-2nd-machine-unless-mount-is-restarted/20645
+            "dir-cache-time": "10s",
+        },
+    }
+
+    extra_options = None
+
+    if r_clone_settings.S3_PROVIDER == S3Provider.MINIO:
+        extra_options = {
+            "s3-provider": "Minio",
+            "s3-region": "us-east-1",
+            "s3-location_constraint": "",
+            "s3-server_side_encryption": "",
```

Comment:
no s3-acl here?

Reply:
Depending on the provider each one has different settings. Apparently with MINIO it is not required. The other two actually require you to explicitly add it.

Reply:
and what does it do?